sdk/dapp-kit/src/hooks/useSuiClient.ts

```diff
@@ -1,16 +1,8 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import type { SuiClient } from '@mysten/sui.js/client';
-import { createContext, useContext } from 'react';
-
-export const SuiClientContext = createContext<
-	| {
-			client: SuiClient;
-			queryKey: (key: unknown[]) => unknown[];
-	  }
-	| undefined
->(undefined);
+import { useContext } from 'react';
+import { SuiClientContext } from '../components/SuiClientProvider.js';
```

Comment:
should this be `.js` ?

Reply:
yeah, the current state of js tooling is very sad.  Typescript and esbuild both don't want to be responsible for changing file extensions during build, meaning you need to use the compiled extension in source... 😭